e2e/jail/jail_test.go

```diff
@@ -154,6 +182,58 @@ func (s *JailTestSuite) TestEventSignal() {
 	response, err := responseValue.ToString()
 	s.NoError(err, "cannot parse result")
 
-	expectedResponse := `{"jsonrpc":"2.0","result":true}`
+	expectedResponse := `{"result":true}`
 	s.Equal(expectedResponse, response)
 }
+
+// TestCallResponseOrder tests for problem in
+// https://github.com/status-im/status-go/issues/372
+func (s *JailTestSuite) TestCallResponseOrder() {
```

Comment:
This test was removed as well for the same reason.

Reply:
I think it was fixed and this test is ok now. It was a problem with locks in a sync send handler in Jail rather than async.